lib/runner.coffee

```diff
@@ -0,0 +1,89 @@
+{Emitter, BufferedProcess} = require 'atom'
```

Comment:
If you're going to these lengths, how would you feel about using `child_process.spawn`?